### PR TITLE
feat: Migrate client data loading from live API to static JSON files

### PR DESCRIPTION
**Pull request by Log234**
- Replaces all five live API fetches (`votes`, `metrics`, `parameters`, `years`, `parties`) with static JSON file requests served from `/assets/election-data/` — no runtime API dependency for data
- Adds `dataVersion` field to `package.json` (pinned to the Lavinia-api release tag) and a `yarn download-data` script that fetches, SHA-256 verifies, and extracts the election-data bundle from the pinned GitHub release
- Injects `APP_VERSION` and `DATA_VERSION` as build-time constants via webpack `DefinePlugin`; `stateIsInvalid()` now uses these instead of a hardcoded version constant, so `localStorage` is automatically busted when either the app or its data bundle changes
- Adds `copy-webpack-plugin` to copy `src/assets/election-data/` into `dist/` so the static files are served in production builds; `src/assets/election-data` is gitignored
- Extends the CI release pipeline: before building, the `test-and-build` job downloads and SHA-256 verifies the corresponding `election-data.zip` from the pinned Lavinia-api release (using GitHub's native `digest` field — no separate `.sha256` asset needed), validates the extracted `manifest.json` version, then uploads the result as a workflow artifact. The `package` job downloads that same artifact rather than re-fetching from GitHub, so the built release always bundles exactly the data that Cypress tested against

## Developer workflow

```bash
yarn download-data   # fetch + verify the pinned election-data bundle into src/assets/election-data/
yarn start           # dev server serves static files directly from src/assets/election-data/
```

## Test plan

- [x] Run `yarn download-data` on a fresh clone — verify files land in `src/assets/election-data/` and `manifest.json` version matches `package.json#dataVersion`
- [x] Run `yarn start` — verify the application loads election data without errors
- [x] Run `yarn build` — verify `dist/assets/election-data/` is populated and the bundle references the correct `DATA_VERSION`

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -23,7 +23,6 @@ jobs:
     env:
       CYPRESS_RECORD_KEY: ${{ secrets.CYPRESS_RECORD_KEY }}
       BUILD_BUILDNUMBER: ${{ github.run_id }}-${{ github.run_attempt }}
-      API_V3: https://api.lavinia.no/api/v3.0.0/
       SWAGGERUI: https://api.lavinia.no/
```
Comment (simendag) [inline .github/workflows/release.yml]: Maybe it makes more sense to link to "data" and the API GitHub? Or maybe even the release?

File: package.json [hunk 2]
```diff
@@ -1,6 +1,7 @@
 {
     "name": "lavinia",
     "version": "2.9.9",
+    "dataVersion": "3.5.0",
```
Comment (simendag) [inline package.json:4]: Do we still load the data into localStorage at all now?

Does the web server control whether to invalidate the assets, or does the client?

I'm not 100% sure I understand how the invalidation aspect here works.

File: package.json [hunk 3]
```diff
@@ -36,17 +38,18 @@
         "bulma-switch": "^2.0.0",
         "bulma-tooltip": "^3.0.2",
         "clean-webpack-plugin": "^4.0.0",
+        "copy-webpack-plugin": "^14.0.0",
```
Comment (simendag) [inline package.json]: I think we may want to stop using the ^ for versions. It would be better if we locked in and patched specific versions; at least then we have some clue what we're patching without checking the lockfile 😅

**Review by simendag — CHANGES_REQUESTED** [review]

File: .github/workflows/release.yml (hunk 1 above)
Comment (Log234) [reply .github/workflows/release.yml]: Oh yeah, I guess the Swagger API won't be relevant much longer. Good point.

File: package.json (hunk 2 above)
Comment (Log234) [reply package.json:4]: We still load the data into localStorage. At the moment the code essentially says if the dataVersion from the client does not match the dataVersion in local storage, fetch everything again. If the version of the client is not compatible with the client version stored in localStorage (minor or major version discrepancy), fetch everything again.

This means that any new data releases will not be used before we release a new client update that bumps the dataVersion in the client. This is a consequence of locking the dataVersion in the package.json file. Alternatively we could have just fetched the latest release available from GitHub.

File: package.json (hunk 3 above)
Comment (Log234) [reply package.json]: Fair enough, I can throw in a commit locking down the versions.

**Comment by simendag** [comment]
We probably need an update to the README.md as part of this change.

**Review by simendag — APPROVED** [review]
Looks good to me, tested locally in a worktree (so clean everything). There might be edge cases with updates, since there's no tests for it, but at least they'll be surprising :-)

I'll approve and assume one of us will update the README.md to reflect the developer workflow either as part of the change or after.
